ghost-cat-app/src/app/View/search-page/search-page.component.ts

```diff
@@ -37,8 +39,47 @@ export class SearchPageComponent implements OnInit {
       new CameraLocation('site006', 40.77956, -110.67389),
       new CameraLocation('site008', 40.77956, -110.77389),
     ];
+    for(let i = 0; i < this.cameraTrapsSelected.length; i++){
+      alert("Current Cameras selected " + this.cameraTrapsSelected[i] + "\n");
```

Comment:
I assume we probably want to get rid of this alert before we push to the website